.ci/openshift-ci/cluster/deployments/relable_selinux.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: restorecon-daemonset
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: restorecon
+  template:
+    metadata:
+      labels:
+        app: restorecon
+    spec:
+      serviceAccountName: kata-deploy-sa
+      hostPID: true
```

Comment:
Out of curiosity: is the hostPID needed so that it can `nsenter --target 1` ?

Reply:
As far as I know, yes.